src/gui/mod.rs

```diff
@@ -19,8 +19,7 @@ use crate::{
     is_drg_pak,
     providers::ModioTags,
     providers::{FetchProgress, ModSpecification, ModStore, ProviderFactory},
-    state::ModProfiles,
-    state::{ModConfig, State},
+    state::{ModConfig, ModData_v0_0_0, State},
```

Comment:
I think this would look better if the import were renamed to not have the version attached: `use ModData_v0_0_0 as ModData`. It would also mean only having to update the version number in the imports rather than every place it's used. Thoughts?

Reply:
Added import alias to avoid having to name the exact versioned struct in other places in the code.